ci/do_ci.sh

```diff
@@ -34,8 +33,8 @@ elif [[ "$1" == "bazel.asan" ]]; then
   cd "${ENVOY_CONSUMER_SRCDIR}"
   echo "Building and testing..."
   # TODO(htuch): This should switch to using clang when available.
-  bazel --batch test ${BAZEL_TEST_OPTIONS} -c dbg --config=asan --test_output=all \
-    --cache_test_results=no @envoy//test/... //:echo2_integration_test
+  bazel --batch test ${BAZEL_TEST_OPTIONS} -c dbg --config=asan @envoy//test/... \
```

Comment:
Yes, but it also refers to the "current" workspace, since that's the name defined in `WORKSPACE`.

Although, now that I tested it, it seems that it doesn't work... I could swear that it did (some time ago), so either they regressed or I'm going crazy. See: https://github.com/bazelbuild/bazel/issues/1866

Anyway, please ignore this comment.